### ISSUE TITLE
Unable to craft using a crafting table

### ISSUE THREAD

**Issue by buchwasa** [pmmp/PocketMine-MP#3889]
### Issue description
I've only tested this with wooden planks trying to make a crafting table in the crafting table, I'm using the bottom left slots.

- Expected result: Be able to craft an item
- Actual result: Transaction fails

### Steps to reproduce the issue
1. Use a crafting table
2. Craft

### OS and versions

* PocketMine-MP: master branch 2e9b2d4aae470021457846d78eda025e2ec702cb but noticed it on ddc0f137e7b4876bc52beb65fb2bfa9edf400cc9
* PHP: 7.4.11
* Server OS: Linux
* Game version: PE/Win10

### Plugins
Issue occurs without plugins

### Crashdump, backtrace or other files
I modified the debug output to include all the failed slots rather than include more lines here than necessary
```
Unhandled inventory action: Unmatched UI inventory slot offset 39, 38, 35, 36
Unhandled InventoryTransactionPacket: HgAAAAKfjQYJBwAKAgAAAAAA+AEnCgIAAAAAAA==
```

**Comment by dktapps** [comment]
Does this occur on releases?

**Comment by buchwasa** [comment]
It's on bleeding edge, not stable

**Comment by dktapps** [comment]
I'm not able to reproduce this.

**Comment by dktapps** [comment]
This doesn't make any sense. The slots are clearly mapped as seen here: https://github.com/pmmp/PocketMine-MP/blob/master/src/network/mcpe/protocol/types/inventory/UIInventorySlotOffset.php#L86-L90
https://github.com/pmmp/PocketMine-MP/blob/master/src/network/mcpe/convert/TypeConverter.php#L199-L202

**Comment by buchwasa** [comment]
Looks like it was a plugin, I had to completely remove it, originally I blacklisted the plugin which apparently had no effect

**Comment by dktapps** [comment]
Fixed by 72ace3951e33075f63ef31f548901e9350ef0c6d.
